en/docs/administer/configuring-keystores-in-wso2-products.md

```diff
@@ -0,0 +1,93 @@
+# Configuring Keystores
+
+After you have [created a new keystore and updated the `client-truststore.jks` file](../../administer/creating-new-keystores), you must update a few configuration files in order to make the keystores work. Note that keystores are used for multiple functions in WSO2 Identity Server, which includes authenticating communication over SSL/TLS, encrypting pass words and other confidential information in configuration files etc. Therefore, you must update the specific configuration files with the updated keystore information. For example, you may have separate keystores for the purpose of encrypting
+passwords in configuration files, and for authenticating communication over SSL/TLS.
+
+The `wso2carbon.jks` keystore file, which is shipped with WSO2 Identity Server, is used as the default keystore for all functions. However, in a production environment, it is recommended to create new keystores with new keys and certificates.
+    
+!!! tip "Before you begin"
+    1.  If you want an easy way to locate all the configuration files that have references to keystores, you can use the grep command as follows:
+
+        1.  Navigate to the `<IS_HOME>/repository/conf/` directory on a command prompt.
+
+        2.  Execute the following command.
+            ```
+            grep -nr ".jks"
+            ``` 
+
+            Note that the configuration files and the keystore files referred to in each file are listed out. Following is an example: 
+
+
+            ```text
+            ./axis2/axis2.xml:260:                <Location>repository/resources/security/wso2carbon.jks</Location>
```

Comment:
These .xml config are no longer valid. Need to update with proper toml based config.

Reply:
Fixed